MODULE.bazel

```diff
@@ -0,0 +1,31 @@
+module(name = "cloe", version = "0.25.0")
+
+# Build dependencies:
+bazel_dep(name = "cmake_configure_file", version = "0.1.0")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+
+# Test dependencies:
+bazel_dep(name = "googletest", version = "1.14.0", repo_name = "gtest")
+
+# Library dependencies:
+bazel_dep(name = "boost", version = "1.83.0.bcr.1")
```

Comment:
provide a link to the PR where this boost module is proposed to added to the bcr

Reply:
Also mark all the modules that are not officially available in the bcr yet, so a user knows for which libraries he has to manually provide a bazel module

Reply:
They are marked in the README, and I don't really want to provide a link to a PR for Boost support, because I do not want to take responsibility for that.

Reply:
Exactly :-D

Reply:
Hey look here: https://github.com/bazelboost/registry